doc/api/deprecations.md

```diff
@@ -700,6 +700,15 @@ Type: Runtime
 
 `REPLServer.turnOffEditorMode()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: Custom inspection function on Objects via .inspect()
+
+Type: Documentation-only
+
+Using a property named `inspect` on an object to specify a custom inspection
+function for [`util.inspect()`][] is deprecated. Use [`util.inspect.custom`][]
+instead.
```

Comment:
It might be helpful to note that, if compatibility is desired, having the method present under both names is *always* going to work?

Reply:
I like the idea but I'm struggling with how to word it in a way that doesn't encourage people to use both if they're only using the Symbol. We don't want to create *more* messages if/when we move to a runtime deprecation. Suggestions welcome.

Reply:
@Trott The thing is, even if we move to a runtime deprecation, the symbol property will be the first one that’s detected, and the string property ignored – so if both are present, the runtime deprecation won’t be emitted… or am I misunderstanding what you’re saying?

Reply:
@addaleax Good point. I've added some text in a separate commit.